packages/flutter/lib/src/material/floating_action_button_theme.dart

```diff
@@ -207,12 +217,13 @@ class FloatingActionButtonThemeData with Diagnosticable {
       extendedIconLabelSpacing: lerpDouble(a?.extendedIconLabelSpacing, b?.extendedIconLabelSpacing, t),
       extendedPadding: EdgeInsetsGeometry.lerp(a?.extendedPadding, b?.extendedPadding, t),
       extendedTextStyle: TextStyle.lerp(a?.extendedTextStyle, b?.extendedTextStyle, t),
+      mouseCursor: t < 0.5 ? a?.mouseCursor : b?.mouseCursor,
     );
   }
 
   @override
   int get hashCode {
-    return hashValues(
+    return hashList(<Object?>[
```

Comment:
`hashValues` is more efficient than `hashList`. lists need to be allocated, which causes GC pressure, etc.

Reply:
`hashValues` takes max 20 arguments. Adding `mouseCursor` increased the amount of arguments to 21.

Reply:
Yes, but while working on the PR for migrating off of `hashValues`, Hixie pointed out that nested `hashValues` is better than `hashList`: https://github.com/flutter/flutter/pull/96109#discussion_r780542232.

Reply:
Thanks, that's interesting. I've updated :)

Reply:
tbh I'm not sure if this NEEDS to use `Object.hash`. It will reduce one conflict in my migration PR tho :p

Reply:
there are so many ways to put this. :) return `hashValues` and use nested `Object.hash`? or nested `hashValues` since the migration is still ongoing? or just keep it as it is now? which way do you prefer to handle it?

Reply:
well, i'd prefer using nested `hashValues` itself, as all the other PRs are doing the same while the migration is ongoing.

Reply:
Alright, done :+1: